package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "n-content-body",
+  "version": "0.0.0",
+  "devDependencies": {
```

Comment:
next-build-tools is required for bottling the release so included to kick that process off
plus have put a verify in using obt.